templates/channel.pug

```diff
@@ -63,7 +63,10 @@ html(lang="en")
               button#newpollbtn.btn.btn-sm.btn-default: span: svg.ch-icon(viewbox="0 0 24 24"): use(href="#ch-icon-ui-poll")
               button#emotelistbtn.btn.btn-sm.btn-default: span: svg.ch-icon(viewbox="0 0 24 24"): use(href="#ch-icon-emote-dead")
               button#chatoptsbtn.btn.btn-sm.btn-default: span: svg.ch-icon(viewbox="0 0 24 24"): use(href="#ch-icon-ui-preferences")
-              button#coolpointsbtn.btn.btn-sm.btn-default(onclick="javascript:showCoolPointsUserPrompt()"): span: svg.ch-icon(viewbox="0 0 24 24"): use(href="#ch-icon-ui-coin-stencil") CoolPoints
+              button#coolpointsbtn.btn.btn-sm.btn-default(onclick="javascript:showCoolPointsUserPrompt()"): span: svg.ch-icon(viewbox="0 0 24 24"): use(href="#ch-icon-ui-coin-stencil")
+                svg.ch-icon.ch-button-icon.ch-cp: use(xlink:href="#ch-icon-ui-coin")
+                span#coinAmt
```

Comment:
nit: this could use a little centering IMO
![image](https://github.com/user-attachments/assets/7cfcc982-0730-4e09-ba3f-c4262cb7ccd3)
![image](https://github.com/user-attachments/assets/09037f90-03d3-4ff0-95b5-80395add98aa)
I achieved the second by just making this wrapping span a flexbox and aligning to center

Reply:
Flexbox works most of the time, but I believe the it would begin to wrap below 1024px, which I think is Bootstrap 3's large breakpoint size. The item's centering might be off due to some default Sync/Cytube CSS for line-height. I'll take another look and see if I can pin that down.

Reply:
Oh yea? I tried it out on a mobile viewport and it looked alright. tried it with 1 CP and 1000000 CP and seemed alright but lmk what you find